.github/workflows/tests.yml

```diff
@@ -40,21 +40,46 @@ jobs:
     steps:
       - name: Get contents from tip of branch on push
         uses: actions/checkout@v4
-
       - name: Set up Python ${{ matrix.python }}
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python }}
-      - name: Install Python dependencies
+      - name: Install scout package
         run: |
           python -m pip install --upgrade pip
           pip install .
       - name: Run tests
         run: python -m unittest discover -p '*_test.py' tests/
 
+  check-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get contents from tip of branch on push
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            scout
+            docs
```

Comment:
fyi - easy way to avoid checking out the full repo w/ ecm_definitions if they are not needed.